src/main/java/org/apache/sysds/runtime/controlprogram/federated/FederationMap.java

```diff
@@ -98,6 +98,29 @@ public boolean isType(FType t) {
 		}
 	}
 
+	// Alignment Check Type
+	public enum AType {
```

Comment:
Suggestion:
Could this be called "AlignType" or something else that is a bit more specific. I read "AType" as "a type" which is a bit too generic :smiley:

Reply:
It was so tempting to name it _AType_ alongside the other enum named _FType_ :rofl: 
I changed the name to _AlignType_ now :+1: